packages/bsky/src/views/index.ts

```diff
@@ -1149,12 +1149,16 @@ export class Views {
       branchingFactor,
       prioritizeFollowedUsers,
       sort,
+      threadTagsBumpDown,
+      threadTagsHide,
     }: {
       above: number
       below: number
       branchingFactor: number
       prioritizeFollowedUsers: boolean
       sort: GetPostThreadV2QueryParams['sort']
+      threadTagsBumpDown: readonly string[]
+      threadTagsHide: readonly string[]
```

Comment:
These get piped around quite a bit, but they are also static to the server.  I wonder if there's room to hoist it up into config for the `Views` class and avoid all the plumbing.